rs_mcai_worker_sdk/src/message/media/mod.rs

```diff
@@ -49,6 +54,28 @@ impl StreamDescriptor {
       index,
       audio_configuration: None,
       image_configuration: Some(ImageConfiguration { filters }),
+      ebu_ttml_live_configuration: None,
+      json_configuration: None,
+    }
+  }
+
+  pub fn new_ebu_ttml_live(index: usize) -> Self {
+    StreamDescriptor {
```

Comment:
Modify stream descriptor with 2 entries :
 - index
 - configuration, which is an enum running on the different configuration possible (image, audio, EBU, JSON)